odoo-shippable/scripts/build-image.sh

```diff
@@ -74,7 +79,11 @@ add_custom_aptsource "${GITCORE_PPA_REPO}" "${GITCORE_PPA_KEY}"
 # Release the apt monster!
 apt-get update
 apt-get upgrade
-apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
+apt-get install -y ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
```

Comment:
You can remove the `-y` for `apt-get install` commands because this image has a environment variable global to set `-y` by default